config/storage.yml

```diff
@@ -1 +1,7 @@
-# Empty file to satisfy ActiveStorage
+local:
```

Comment:
Getting this error when running sneakers, when there is no valid configuration in this file:

```
rails aborted!
KeyError: Missing configuration for the local Active Storage service. Configurations available for the  services.
```